### PR TITLE
Added solution

### PR DESCRIPTION
**Pull request by IlyaGrynyshyn**

### CODE REVIEW

File: app/test_main.py [hunk 1]
```diff
@@ -1,3 +1,74 @@
...
+            ),
+            (
+                (28, 4), (28, 1), TypeError
+            )
+        ]
+    )
+    def test_get_human_age_invalid_error(
+            self,
+            cat_age: int,
+            dog_age: int,
```
Comment (Dimosphen1) [inline app/test_main.py:70]: Change your type annotation to indicate that incoming arguments could be `Any` by their type according to the test setup, it is not guaranteed that they would be integers.

File: app/test_main.py [hunk 2]
```diff
@@ -1,3 +1,74 @@
...
+            error: Exception
```
Comment (Dimosphen1) [inline app/test_main.py:71]: Indicate that the `error` should be not an instance of `Exception`, but rather a subclass of one.

File: app/test_main.py [hunk 3]
```diff
@@ -1,3 +1,74 @@
...
+            (
+                (),
+                (),
+                TypeError
+            ),
+            (
+                28, (), TypeError
+            ),
+            (
+                (28, 4), (28, 1), TypeError
```
Comment (Dimosphen1) [inline app/test_main.py:63]: Add tests with datatypes other than tuples for this task (e.g. str, dict, set, list, e.g.).

File: app/test_main.py [hunk 4]
```diff
@@ -1,3 +1,74 @@
...
+                14,
+                14,
+                [0, 0]
+            ),
+            (
+                15, 15, [1, 1]
+            ),
+            (
+                23, 23, [1, 1]
+            ),
```
Comment (Dimosphen1) [inline app/test_main.py:20]: There is no need for such broad spacing, it affects the code perception. Reformat your code style as follows:
```suggestion
        [
            (14, 14, [0, 0]),
            (15, 15, [1, 1]),
            (23, 23, [1, 1]),
```
Apply this to `test_get_human_age_invalid_error` as well.

File: app/test_main.py [hunk 5]
```diff
@@ -1,3 +1,74 @@
...
+            (
+                0, 0, [0, 0]
+            )
+        ]
+    )
+    def test_get_human_age(
+            self, cat_age: int,
+            dog_age: int,
+            expected_age: list
```
Comment (Dimosphen1) [inline app/test_main.py:47]: Place each argument on a new line, including `self`. Besides that, indicate that you pass either `int` or `float` for `dog_age` and `cat_age`.

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested.

**Review by pavlejviki — APPROVED** [review]
Well done

**Pull request closed without merging**